website/docs/quick-start-guide.md

```diff
@@ -20,7 +20,8 @@ Hudi works with Spark-2.4.3+ & Spark 3.x versions. You can follow instructions [
 
 | Hudi            | Supported Spark 3 version                       |
 |:----------------|:------------------------------------------------|
-| 0.12.0 - 0.13.x | 3.3.x (default build), 3.2.x, 3.1.x             |
+| 0.13.x          | 3.3.x (default build), 3.2.x, 3.1.x             |
```

Comment:
Could you add this entry (0.13.x) to the 0.13.x version of "Spark Guide" page too?

Reply:
@yihua  this is the spark guide page. The name of this file is misleading though. Refer the title - https://github.com/apache/hudi/pull/9603/files#diff-cb3da508fe853d0882b7057d91605d974a76bf55e9acc5508ccb813bcc015fdcR2

Reply:
done